public/app/plugins/datasource/loki/querybuilder/parsing.ts

```diff
@@ -1,5 +1,4 @@
 import { SyntaxNode } from '@lezer/common';
-import { BinModifiers, OnOrIgnoring } from '@prometheus-io/lezer-promql';
```

Comment:
This is the real bug 🤦 Importing constants from the wrong package

Reply:
Autocomplete is a double-edge sword.